tests/core/test_overdrive.py

```diff
@@ -513,14 +513,17 @@ def test_book_info_to_circulation(self):
         )
 
     def test_book_info_to_circulation_advantage(self):
-        # Overdrive Advantage accounts derive different information
-        # from the same API responses as regular Overdrive accounts.
+        # Overdrive Advantage accounts (a.k.a. "child" or "sub" accounts derive
+        # different information from the same API responses as "main" Overdrive
+        # accounts.
```

Comment:
I will have verify. Good catch.

Reply:
I have definitely verified that we get both the main and sub accounts when calling the context of a parent that has associated sub accounts.  As far as calling the context of the sub account,  I have not verified what we get back in the case of a child account.

Reply:
I will look into what we get back when calling in the context of an child account.

Reply:
I'm not sure how to set this test up.  But that it should be okay since our handling of the child accounts hasn't changed except that we are excluding the counts if they are shared.